operators/hack/api-docs/templates/pkg.tpl

```diff
@@ -0,0 +1,39 @@
+{{- define "packages" -}}
+// Generated documentation. Please do not edit.
+[id="{p}-reference"]
+== Reference
```

Comment:
Should this be `API Reference` or `CRD Reference` because its also used for the index?
![Screenshot 2019-08-12 at 11 23 19](https://user-images.githubusercontent.com/697790/62855897-b4542900-bcf3-11e9-835a-e3d14a029ae5.png)